### DISCUSSION TITLE
Can access the localhost but not the IP Address

### DISCUSSION THREAD

**Discussion by aliakbarjausyani** [leokhoa/laragon#517] [General]
So I was trying to setup my Laravel project to a client's PC remotely using Laragon with Nginx. I install the Laragon, uncheck all the services except nginx, took my Laravel folder to the www folder, then start the services. It works normally on my PC, I can access the project by localhost, the custom URL by Laragon, and by the IP Address.
But not on my client's PC. The localhost and the custom URL can be accessed normally but not by it's own IP Address. The only reason I can think of is the firewall. I have added the inbound rules for the nginx both TCP and UDP but still no avail. The client said they used McAfee and Fortigate for their security, and as of right now they are trying to disabled them. Is there any possible answer to this beside the McAfee or Fortigate?

**Comment by Andres-M-Rios** [comment]
Fortigate? Ya you need to set up rules on that. Umm, good luck 🤞 Andes M. ***@***.*** Jun 9, 2023, at 23:09, Ali Akbar Jausyani ***@***.***> wrote:﻿
So I was trying to setup my Laravel project to a client's PC remotely using Laragon with Nginx. I install the Laragon, uncheck all the services except nginx, took my Laravel folder to the www folder, then start the services. It works normally on my PC, I can access the project by localhost, the custom URL by Laragon, and by the IP Address.
But not on my client's PC. The localhost and the custom URL can be accessed normally but not by it's own IP Address. The only reason I can think of is the firewall. I have added the inbound rules for the nginx both TCP and UDP but still no avail. The client said they used McAfee and Fortigate for their security, and as of right now they are trying to disabled them. Is there any possible answer to this beside the McAfee or Fortigate?
